src/app/mat-main-nav/mobile-menu/mobile-menu.component.html

```diff
@@ -0,0 +1,25 @@
+<mat-accordion displayMode="flat" *ngFor="let menuItem of menuItems$">
+    <mat-expansion-panel *ngIf="menuItem.menuItems">
+        <mat-expansion-panel-header collapsedHeight="48px" expandedHeight="48px">
+            {{menuItem.name}}
+        </mat-expansion-panel-header>
+        <mat-nav-list *ngFor="let item of menuItem.menuItems; let i = index; let last = last">
+            <a mat-list-item *ngIf="item.type === 0" routerLink="['{{ item.route }}']">{{item.name}}</a>
+            <a mat-list-item *ngIf="item.type === 1" [href]="hostURI + item.route">{{item.name}}</a>
+            <!-- show divider every 4th element using modulus -->
+            <mat-divider *ngIf="(i === 3) || (i > 0 && (i+1) % 4 === 0 && !last)"></mat-divider>
```

Comment:
is it imperative to use the mat-divider with a complex if statement? might be easier to achieve the border with css + nth-child...

Reply:
very very true. nth-child css

Reply:
done